src/Domain/CoordinateMaps/Frustum.hpp

```diff
@@ -335,6 +335,7 @@ class Frustum {
   bool with_equiangular_map_{false};
   bool is_identity_{false};
   Distribution zeta_distribution_ = Distribution::Linear;
+  double zeta_distribution_value_{std::numeric_limits<double>::signaling_NaN()};
```

Comment:
also add to `==` operator (careful with the `NaN` value)

Reply:
When adding to the `==` operator, it seems to break the test in `Test_Domain.cpp` and I'm not exactly sure why. I code --diff'd the 2 domains being compared and they seem to be the same (from the test output). Does the `SerializedDomain.h5` need to be updated?

Reply:
I think this is because `inner_radius_` is unpacked to NaN from the data written by the old code. I suggest you set the `zeta_distribution_value_` only if the distribution is set to logarithmic, because only then do you actually need it. (both in the constructor and in pup)